apps/test-suite/tests/FileSystemNext.ts

```diff
@@ -456,6 +456,148 @@ export async function test({ describe, expect, it, ...t }) {
       addAppleAppGroupsTestSuiteAsync({ describe, expect, it, ...t });
     }
   });
+
+  describe('Exposes file handles', () => {
+    it('Allows opening files', () => {
+      const src = new File(testDirectory + 'file.txt');
```

Comment:
Please, we really need `new File(testDirectory, 'file.txt')` API 😅

Reply:
we have it already, I'm just a dummy who copies old test functions around